app/models/region.rb

```diff
@@ -89,7 +89,11 @@ def cohort_analytics(period:, prev_periods:)
   end
 
   def dashboard_analytics(period:, prev_periods:, include_current_period: true)
-    DistrictAnalyticsQuery.new(self, period, prev_periods, include_current_period: include_current_period).call
+    if facility_region?
```

Comment:
Oh, so, since `FacilityAnalyticsQuery` was never being responded to, we never saw the user details?

Reply:
Ha, exactly. Every region thought its dashboard analytics was supposed to be district analytics - so when we started _using_ regions in reports, the facility reports borked.